docker/templates/Dockerfile.j2

```diff
@@ -147,7 +147,25 @@ ADD pipeline/default.conf pipeline/logstash.conf
 RUN chown --recursive logstash:root config/ pipeline/
 # Ensure Logstash gets the correct locale by default.
 ENV LANG={{ locale }} LC_ALL={{ locale }}
-ADD env2yaml/env2yaml /usr/local/bin/
+
+# Copy over the appropriate env2yaml artifact
+COPY env2yaml/env2yaml-amd64 env2yaml/
+COPY env2yaml/env2yaml-arm64 env2yaml/
+
+RUN env2yamlarch="$(arch)"; \
```

Comment:
We should address the comment made here https://github.com/docker-library/official-images/pull/16520#issuecomment-2043738623, to use the pkg manager to get the architecture.